### PR TITLE
bytesize hgtseq

### PR DESCRIPTION
**Pull request by FranBonath**

### CODE REVIEW

File: markdown/events/2023/bytesize_hgtseq.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: 'Bytesize: nf-core/hgtseq'
+subtitle: Francesco Lescai - Department of Biology and Biotechnology, University of Pavia
+type: talk
+start_date: '2023-03-21'
+start_time: '13:00 CET'
```
Comment (lescai) [inline markdown/events/2023/bytesize_hgtseq.md:6]:
```suggestion
start_time: '13:30 CET'
```

File: markdown/events/2023/bytesize_hgtseq.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+end_date: '2023-03-21'
+end_time: '13:30 CET'
```
Comment (lescai) [inline markdown/events/2023/bytesize_hgtseq.md:8]:
```suggestion
end_time: '14:00 CET'
```

**Review by lescai — CHANGES_REQUESTED** [review]
just fixed the start time as discussed :)

**Comment by FranBonath** [comment]
yes, @lescai. Sorry about this!

**Review by lescai — APPROVED** [review]

File: markdown/events/2023/bytesize_hgtseq.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+Join us for our **weekly series** of short talks: **“nf-core/bytesize”**.
+
+Just **15 minutes** + questions, we will be focussing on topics about using and developing nf-core pipelines.
+These will be recorded and made available at <https://nf-co.re>
+It is our hope that these talks / videos will build an archive of training material that can complement our documentation. Got an idea for a talk? Let us know on the [`#bytesize`](https://nfcore.slack.com/channels/bytesize) Slack channel!
+
+## Bytesize: nf-core/hgtseq
+
+This week, Francesco Lescai ([@lescai](https://github.com/lescai)) is presenting us the nf-core pipeline hgtseq. Nf-core/hgtseq is a bioinformatics best-practice analysis pipeline built to investigate horizontal gene transfer from NGS data.
```
Comment (mashehu) [inline markdown/events/2023/bytesize_hgtseq.md:23]:
```suggestion
This week, Francesco Lescai ([@lescai](https://github.com/lescai)) is presenting us the nf-core pipeline hgtseq. nf-core/hgtseq is a bioinformatics best-practice analysis pipeline built to investigate horizontal gene transfer from NGS data.
```

**Review by mashehu — APPROVED** [review]

File: markdown/events/2023/bytesize_hgtseq.md (hunk 3 above)
Comment (FranBonath) [reply markdown/events/2023/bytesize_hgtseq.md:23]: Really?

**Pull request merged**